projects/plugins/boost/app/assets/src/js/features/critical-css/error-description/error-description.tsx

```diff
@@ -15,6 +14,14 @@ import MoreList from '../more-list/more-list';
 import styles from './error-description.module.scss';
 import Suggestion from '../suggestion/suggestion';
 import { CriticalCssErrorDescriptionTypes, FormattedURL } from './types';
+import { useRegenerateCriticalCssAction } from '../lib/stores/critical-css-state';
+import { useNavigate } from 'react-router-dom';
+
+function stripCacheParams( url: string ): string {
+	const urlObj = new URL( url );
+	urlObj.searchParams.delete( 'donotcachepage' );
+	return urlObj.toString();
+}
```

Comment:
A comment explaining why stripping is necessary would be useful

Reply:
Fixed in 10298d4936.